harvester/files/migrations/0003_merge_0002_finished_at_0002_youtube_api_resource.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.2.20 on 2023-10-18 12:51
+
+from django.db import migrations
```

Comment:
These migration conflicts are problematic and can't be solved this way.

You can run: python manage.py migrate files 0001 (this will bring migration state to before the conflict)

Then throw away all migrations you've made so far and re-run: python makemigrations -n youtube_api_resource

Commit those migrations (should be 0003_youtube_api_resource) and roll-out on development (where 0002_finished_at has already run) will be fine.